docs/workgroup/charter.rst

```diff
@@ -0,0 +1,136 @@
+===================================
+CPython Documentation Working Group
+===================================
+
+Purpose & Common Goals
+----------------------
+
+This workgroup will support efforts to improve
+and maintain CPython's documentation through curation,
+internationalization, and special projects. Its focus will also include
+tutorials, how-to documents, the devguide and other documentation in support
+of the PSF's mission to expand the global Python userbase.
+Quality and effectiveness will be key goals as we build a
+diverse team of community members, documentarians, and core developers.
+
+Things the workgroup could do:
+
+- Develop the governance model for docs
+- Be the steering council for docs
+- Be the authors of doc guidelines
+- Be the authors/editors of docs
+- Request grant funding from PSF, if needed
+- Form an editorial board for docs including core devs, educators, and
+  documentarians in the Python community
+
+Active Time
+-----------
+
+This workgroup will be active indefinitely while supported by the Python
+Steering Council.
+
+Core Values & Internal Governance
+---------------------------------
+
+The core values for this group are:
+
+- Respect
+- Support
+- Positive Atmosphere
+- Constructiveness
+- Integrity
+
+The workgroup adopts the `PSF Code of Conduct <https://www.python.org/psf/codeofconduct/>`_.
+Any action by a workgroup member, as decided by a majority of the group,
+that violates the principles in the Code of Conduct will result in that member
+being removed from the workgroup.
+
+Rules & Guidelines
+------------------
+
+- At any time, the Python Steering Council may, at its discretion, add any number
+  of non-voting observers to the working group to observe and participate in
+  discussion.
```

Comment:
"observers to ... observe" feels redundant, perhaps just leave out from "group"
```suggestion
  of non-voting observers to the working group.
```

Reply:
```suggestion
  of non-voting observers to the working group to participate in discussion.
```

I agree with your point about "observe", but I think it does add value to clarify that they may "participate in discussion", just not vote (as opposed to simply observe), so I suggest the above.